examples/LoginForm.re

```diff
@@ -56,15 +56,20 @@ module LoginForm = {
   let validators = [EmailField.validator, PasswordField.validator];
 };
 
-module LoginFormContainer = Formality.Make(LoginForm);
+module LoginFormHook = Formality.Make(LoginForm);
 
-let component = React.statelessComponent(__MODULE__);
-let make = _ => {
-  ...component,
-  render: _ =>
-    <LoginFormContainer
-      initialState={email: "", password: "", rememberMe: false}
-      onSubmit={(state, form) => {
+[@react.component]
+let make = () => {
+  let initialState =
+    React.useMemo1(
```

Comment:
Is this use of useMemo useful here? Why not extract initial state outside of the component?

Reply:
If you're doing this to make `initialState` private without adding an rei we support the block syntax for components. Also `useMemo0` is equivalent to `useMemo1` with an empty array.

```reason
[@react.component]
let make = {
  let initializationStuff = 1;
  (~prop) => ...
};
```

Reply:
Ohh very nice! TBH I used it this way for no particular reason, just getting myself familiar with hooks. Will address, thanks for suggestions!